packages/redux/src/bags/reducer.ts

```diff
@@ -190,6 +195,12 @@ const bagOperations = (
   }
 };
 
+const bagPromocodes = reducerFactory(
+  'SET_BAG_PROMOCODES',
+  INITIAL_STATE.bagPromocodes,
+  actionTypes,
+);
+
 export const entitiesMapper = {
```

Comment:
Take a look at the entitiesMapper for the `LOGOUT_SUCCESS` action which is not resetting the `bagPromocodesInformation` entities (and the `bagOperations` entities as well which I believe it should). Perhaps it would be best to extract the reset logic to a common function and use that function in `RESET_BAG_ENTITIES` and `LOGOUT_SUCCESS` since they should do the same.

Reply:
I created a function called `resetBagEntities` and I made something similar to what we do on the checkout reducer.
Please review this.

Reply:
That is great, thanks!